lib/trino-parquet/src/main/java/io/trino/parquet/ParquetReaderOptions.java

```diff
@@ -15,17 +15,20 @@
 
```

Comment:
Please shorten commit title to
`Add configurable property for max batch size to parquet reader`
The general guideline is that we try to avoid the commit title from wrapping on GitHub.